bin/bot/src/discord/slash/bank.rs

```diff
@@ -33,6 +34,12 @@ pub fn register(command: &mut CreateApplicationCommand) -> &mut CreateApplicatio
                         .required(true)
                 })
                 .allow_public()
+        }).create_option(|option| {
+            option
+            .name("group_balance")
+            .description("View the Group balance")
```

Comment:
group lowercase?

Reply:
imo should do brodsky as the member to see the group fund

Reply:
i just did!